.spelling

```diff
@@ -183,6 +183,7 @@ memoizing
 metadata
 minikube
 mutex
+mutexes
```

Comment:
For reference, "mutexes" appears to be preferred over "mutices": https://english.stackexchange.com/q/37455/527544